modules/admin_manual/pages/configuration/files/manual_file_locking.adoc

```diff
@@ -3,38 +3,76 @@
 
 == Introduction
 
-Starting with ownCloud Server 10.5 a new enhancement is provided for content collaboration. Manual file locking allows users to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out). 
+Starting with ownCloud Server 10.5 a new enhancement is provided for content collaboration. Manual file locking allows users, if enabled, to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out). 
 
 The feature builds on the WebDAV Locks backend which has been introduced with Server 10.1 and is now available in the ownCloud Web Interface. All storages are supported as locking takes place on the WebDAV level. The locks will only be available via ownCloud, not when a user works directly on the storage. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they can not make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).
 
-== Installation
+== Enable or Disable the UI Component
 
-By default, locks set in the web interface will expire after 30 minutes. The maximum lock time by default is 24 hours.
+NOTE: The _user-facing_ components in the web interface are disabled by default because this feature allows users to lock other user's files *exclusively*. Even the owner of the file can't unlock them, only the locking user can unlock or if the lock expires.
```

Comment:
```suggestion
NOTE: The _user-facing_ components in the web interface are disabled by default because this feature allows users to lock other users' files *exclusively*. Even the owner of the file can't unlock them, only the locking user can unlock until the lock expires.
```